### ISSUE TITLE
Role doesn't install specific package version on RedHat OS

### ISSUE THREAD

**Issue by bigstinky86** [UnderGreen/ansible-role-mongodb#202]
When specify exact package version to install, for example 3.4.22 ansible play fail with error

```
TASK [undergreen.mongodb : Install MongoDB package] *****************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "No package matching 'mongodb-org-server=3.4.22' found available, installed or updated", "rc": 126, "results": ["No package matching 'mongodb-org-server=3.4.22' found available, installed or updated"]}

```

**Comment by bigstinky86** [comment]
PR was created #203

**Comment by UnderGreen** [comment]
Fixed in #203.
